adcircpy/model/tpxo.py

```diff
@@ -114,20 +115,26 @@ def _assert_vertices(self, vertices):
         assert vertices.shape[1] == 2, msg
 
     @staticmethod
-    def _fetch_tpxo_file(prefix, file):
+    def _fetch_tpxo_file(prefix: str, file: str):
         url = "https://www.dropbox.com/s/uc44cbo5s2x4n93/"
         url += "h_tpxo9.v1.tar.gz?dl=1"
 
-        def query_yes_no(question, default="yes"):
-            """Ask a yes/no question via raw_input() and return their answer.
-            "question" is a string that is presented to the user.
-            "default" is the presumed answer if the user just hits <Enter>.
-                It must be "yes" (the default), "no" or None (meaning
-                an answer is required of the user).
-            The "answer" return value is one of "yes" or "no".
+        def query_yes_no(question: str, default: str = "yes") -> bool:
```

Comment:
I like using type hinting because PyCharm will use it to run through usages and warn of type conflicts, plus it's explanatory

Reply:
I like the type hinting as well, please continue to add them as appropriate